src/WCSimWCTrigger.cc

```diff
@@ -406,6 +409,210 @@ void WCSimWCTriggerBase::FillDigitsCollection(WCSimWCDigitsCollection* WCDCPMT,
 }
 
 
+void WCSimWCTriggerBase::FillDigitsCollectionNoTrigger(WCSimWCDigitsCollection* WCDCPMT, bool remove_hits, TriggerType_t save_triggerType)
```

Comment:
As far as I can see the only difference is that no offset is added to 
the digitized time.
We should really ask Trevor because he coded this part and I just have 
to submit it first in order to proceed with the submission of the 
radioactivity part. I am happy with removing this method and using the 
original one though.


On 06/06/16 10:38, tdealtry wrote:
>
> In src/WCSimWCTrigger.cc 
> <https://github.com/WCSim/WCSim/pull/168#discussion_r65862218>:
>
> > @@ -406,6 +409,210 @@ void WCSimWCTriggerBase::FillDigitsCollection(WCSimWCDigitsCollection* WCDCPMT,
> >  }
> >
> >
> > +void WCSimWCTriggerBase::FillDigitsCollectionNoTrigger(WCSimWCDigitsCollection* WCDCPMT, bool remove_hits, TriggerType_t save_triggerType)
>
> A bit of a naive question, but what are the differences between this & 
> FillDigitsCollection()? They look very similar. Is there a way of 
> combining them into a single method?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/WCSim/WCSim/pull/168/files/c96fb17074977c1f63dabf0d22e13c419d99e1ff#r65862218>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ANMXw81VVv8v7zMxvZWDzdLyMCJti72Wks5qI-qBgaJpZM4IutXq>.
>

-- 

_____________________________________________________________
Federico Nova         office: R1 2.89
phone: 01235 44 6980
email:  federico.nova@stfc.ac.uk
_____________________________________________________________
Particle Physics Department            Rutherford Appleton Laboratory
Harwell, Didcot, Oxford, OX11 0QX, UK
_____________________________________________________________